backend/controllers/applicationController.py

```diff
@@ -66,6 +66,30 @@ def registerApplication():
     return abort(400)
 
 
+@application.route("/", methods=["PUT"])
+@requiresUser
+def updateApplication():
+    if request.is_json:
+        ctx = _request_ctx_stack.top
+        user = ctx.user
+        if not user:
```

Comment:
Isnt this redundant as requiresUser fails if there isnt a user?

Reply:
You're right! I'll remove it